pandas/core/accessor.py

```diff
@@ -280,9 +282,7 @@ def decorator(accessor):
 
 
 @doc(
-    _register_accessor,
-    klass="DataFrame",
-    others="register_series_accessor, register_index_accessor",
+    _register_accessor, klass="DataFrame",
```

Comment:
Is black formatting this in three lines, even if you write it in one? I think with one would be clearer.

Reply:
It is in one line.

Reply:
I mean this seems better:
```
@doc(_register_accessor, klass="DataFrame")
def register_dataframe_accessor(name):
   
```